src/ui/ResultList/ResultList.ts

```diff
@@ -239,29 +229,30 @@ export class ResultList extends Component {
     fieldsToInclude: ComponentOptions.buildFieldsOption({ includeInResults: true }),
 
     /**
-     * Specifies the layout to use when displaying results in this `ResultList` (see
+     * The layout to use when displaying results in this `ResultList` (see
      * [Result Layouts](https://developers.coveo.com/x/yQUvAg)). Specifying a value for this option automatically
-     * populates a [`ResultLayout`]{@link ResultLayout} component with a switcher for the layout.
+     * populates a [`ResultLayoutSelector`]{@link ResultLayoutSelector} component with a switcher for the layout.
      *
      * For example, if there are two `ResultList` components in the page, one with its `layout` set to `list` and the
      * other with the same option set to `card`, then the `ResultLayout` component will render two buttons respectively
      * entitled **List** and **Card**.
      *
      * See the [`ValidLayout`]{@link ValidLayout} type for the list of possible values.
      *
-     * Default value is `list`.
+     * @examples card
      */
     layout: ComponentOptions.buildStringOption<ValidLayout>({
       defaultValue: 'list',
       required: true
     }),
 
     /**
-     * Whether to scroll back to the top of the page whenever the end-user interacts with a facet or with the pager.
-     *
-     * Default: `true`
+     * Whether to scroll back to the top of the page whenever the end-user interacts with a non-dynamic facet.
```

Comment:
Sorry if I was unclear, but it will scroll to top on `deferredQuerySuccess` wether it's a dynamic facet select or any other interaction.